src/dmd/link.d

```diff
@@ -220,9 +220,14 @@ public int runLINK()
             }
             for (size_t i = 0; i < global.params.libfiles.dim; i++)
             {
-                cmdbuf.writeByte(' ');
-                cmdbuf.writestring("/DEFAULTLIB:");
-                writeFilename(&cmdbuf, global.params.libfiles[i]);
+                // The `/DEFAULTLIB:` argument cannot be empty
+                const f = global.params.libfiles[i];
+                if (f && strlen(f) > 0)
+                {
+                    cmdbuf.writeByte(' ');
+                    cmdbuf.writestring("/DEFAULTLIB:");
+                    writeFilename(&cmdbuf, f);
+                }
```

Comment:
This looks like it belongs in a separate PR?

Reply:
I could do that, but I don't have a test to verify it.  This is PR has the logic necessary to create a test that actually benefits from the `defaultlib=` (i.e. no defaultlib).  So, I think they need to go together.

Reply:
Should clearly set up the code so that no default lib has `defaultlib` set to null, not a 0 length string.